pkg/services/sqlstore/migrations/ualert/ualert.go

```diff
@@ -226,3 +226,16 @@ func (u *upgradeNgAlerting) updateAlertmanagerFiles(orgId int64, migrator *migra
 func (u *upgradeNgAlerting) SQL(migrator.Dialect) string {
 	return codeMigration
 }
+
+func CheckLegacyAlertingEnabled(mg *migrator.Migrator) {
+	if mg.Cfg.Raw == nil {
+		return
+	}
+	// This check is kept to prevent users that upgrade to Grafana 11 with the legacy alerting enabled. This should prevent them from accidentally upgrading without migration to Unified Alerting.
+	alerting := mg.Cfg.Raw.Section("alerting")
+	enabled, err := alerting.Key("enabled").Bool()
+	if err == nil && enabled {
+		mg.Logger.Error("Option [alerting].enabled cannot be true. Legacy alerting is no longer supported. Delete the option and use configuration '[unified_alerting].enabled' to enable Grafana Alerting and start from the scratch. Refer to documentation about migration to the new alerting (https://grafana.com/docs/grafana/latest/alerting/set-up/migrating-alerts)")
```

Comment:
@brendamuir can you review the text here? It will be shown in the logs for users who upgrade to Grafana 11 but are still using legacy alerting in their configuration file.